.env.example

```
@@ -133,7 +187,7 @@ MAILER_URLPATHS_EMAIL_CHANGE="/auth/v1/verify"
 
 ## Email auth
 ENABLE_EMAIL_SIGNUP=true
-ENABLE_EMAIL_AUTOCONFIRM=false
+ENABLE_EMAIL_AUTOCONFIRM=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_EMAIL_AUTOCONFIRM key should go before the ENABLE_EMAIL_SIGNUP key

(UnorderedKey)